### DISCUSSION TITLE
Kaido Server Subtitle Issue

### DISCUSSION THREAD

**Discussion by Blitz2224** [siennaXD/SaikouTV#5] [General]
kaido server is one the best in terms of picture quality , and the 1080p episodes lookes more sharp and vibrant on Kaido server but the problem is there is not subtitles on this server , not a single anime have subtitle in this server . it would be a big help if you could please resolve it in the next update . 

thanku

**Comment by danyev3** [comment]
Website scrapers get updated when the main saikou app gets updated, this is only a fork for android tv and doesn't aim to improve upon the app it branches from

**Reply by Blitz2224** [reply]
can you keep it simple please

**Reply by danyev3** [reply]
This app is a modified version of https://github.com/saikou-app/saikou and only add support for Android TV. The servers get updated when they get updated in the original project, so you have to ask there or on the discord server

**Reply by Blitz2224** [reply]
but the main app dont have this problem , it is only happening on tv version
